src/local/butler/scripts/setup.py

```diff
@@ -37,6 +37,20 @@
 THREAD_DELAY = 30.0
 """
 
+SYZKALLER_TEMPLATE = """MAX_FUZZ_THREADS = 1
+MAX_TESTCASES = 4
+FUZZ_TEST_TIMEOUT = 4800
+TEST_TIMEOUT = 65
```

Comment:
Is it this long needed to reproduce testcase ? Ideally this should be under 20 sec, otherwise regression testing, etc will be very slow.

Reply:
There are two different modes for syzkaller repro'ing, one using syz-repro that minimizes and repro (this takes time), and another using syz-execprog that only repro (this is fast). So I will change it to 20 and we will assume this will run the fast method and that minimization will be its own task.

Reply:
20 seconds may not be enough for repro because the phone has to reboot.  Plus if we don't disable ramdump you're going to need like 10 minutes,

Reply:
Ok so I will set it for 600 seconds.